### ISSUE TITLE
possible pseudogene alignment

### ISSUE THREAD

**Issue by marleysizzy** [liulab-dfci/TRUST4#284]
Does your Trust4 reference file for the human genome include pseudogenes? If not, do you have any idea on how I could include those genes in?

**Comment by mourisl** [comment]
Yes, the reference file for the CDR3 annotation step is based on IMGT. It does include some pseudogenes, like IGHV1-68. But it might be incomplete. If you have more pseudogene want to add to the list, you can put them into the IMGT+C.fa file, and add the IMGT gaps. If you don't have the gap information, you can remove all the gaps ("." symbol) in the IMGT+C.fa file, and TRUST4 will use motif information to identify CDR3 instead of IMGT coordinate system. Hope this helps.
